docs/Bitbucket.md

```diff
@@ -0,0 +1,169 @@
+| `pipelines.<start-condition>.step.caches` | ✅ | Step-level dependencies downloaded from external sources (Docker, Maven, PyPi for example) which will be able to be re-used in later Bitbucket pipeline steps. Caches that are set at step level (or though the top-level `definition.cache.<name> property) are translated in the corresponding Buildkite pipeline utilising the [cache-buildkite-plugin](https://github.com/buildkite-plugins/cache-buildkite-plugin) to store the downloaded dependencies for re-use between Buildkite builds. |
+| `pipelines.<start-condition>.step.condition` | ✅ | The configuration to prevent a Bitbucket pipeline step from running unless the specific conditional is met. Translated to an inline conditional (`if`) within the corresponding Buildkite pipelines' command step's `commands` - based on a `git diff` of the base branch. |
+| `pipelines.<start-condition>.step.condition.changeset.includePaths` | ⚠️ | The specific file (or files) that need to be detected as changed for the `condition` to apply based. This can be set as specific files - or wildcards that match multiple files in a specific directory/directories. <br/><br/> In cases where many paths are needed to be polled for changes - utilising the [monorepo-diff-buildkite-plugin](https://github.com/buildkite-plugins/monorepo-diff-buildkite-plugin) is recommended; watching for specific folders/files and uploading resultant [Dynamic pipelines](https://buildkite.com/docs/pipelines/defining-steps#dynamic-pipelines) upon diff detection. |
+| `pipelines.<start-condition>.step.clone` | ⚠️ | Clone options for a specific step of a Bitbucket pipeline. The majority of these options should be set on a Buildkite agent itself via [configuration](https://buildkite.com/docs/agent/v3/configuration) of properties such as the clone flags (`git-clone-flags`, `git-clone-mirror-flags` if utilising a Git mirror), fetch flags (`git-fetch-flags`) - or changing the entire checkout process in a customised [plugin](https://buildkite.com/docs/plugins/writing) overriding the default agent `checkout` hook. Sparse checkout options are supported (with the `sparse-checkout` sub-property) |
+| `pipelines.<start-condition>.step.clone.sparse-checkout` | ✅ | Sparse-checkout option for a Bitbucket pipeline step. Translated to utilising the [sparse-checkout-buildkite-plugin](https://github.com/buildkite-plugins/sparse-checkout-buildkite-plugin). <br/><br/> Sparse-checkout properties of `code-mode`, `enabled` and `patterns` will be trasnlated to the respective properties within the mentinoed plugin's configuration. |
```

Comment:
You already mention that this key is supported in the previous entry, I don't think that this is necessary

Reply:
Gotten rid of this row in latest - aligning with the top-level `clone`'s desc 👍